config/webpack.config.prod.js

```diff
@@ -401,6 +402,42 @@ module.exports = {
               'sass-loader'
             ),
           },
+          // SVGR is a tool that converts SVG files into React components that you can use directly in JXS.
+          {
+            test: /\.svg$/,
```

Comment:
@mauriguz did you tested - svgs are working without any issues ?
in both cases where they're used as images and as backgrounds ?